code/modules/examine/examine.dm

```diff
@@ -34,7 +34,7 @@
 
 // Quickly adds the boilerplate code to add an image and padding for the image.
 /proc/desc_panel_image(var/icon_state)
-	return "\icon[description_icons[icon_state]][EXAMINE_PANEL_PADDING]"
+	return "[bicon(description_icons[icon_state])][EXAMINE_PANEL_PADDING]"
```

Comment:
This is specifically the change that seems to have broken the examine tab, at least wrt wall deconstruction icons